test/__snapshots__/loader.test.js.snap

```diff
@@ -347,6 +347,16 @@ Failed to parse source map: \\"ftp://exampleurl.com\\" URL is not supported",
 ]
 `;
 
+exports[`source-map-loader should skip SourceMaps for suppressed resources: css 1`] = `
+"with SourceMap
+//#sourceMappingURL=external-source-map.map
+// comment"
+`;
```

Comment:
Here the interesting problem, if you don't remove this comment, it creates warning(s) in devtools, we should provide ability to keep or remove `sourceMappingURL` comment. How I see it:
```
skipSourceMappingURL: (sourceMappingURL, resourcePath) => { 
  if (/test\.js$/.test(resourcePath)) { 
    return 'remove';
  }
  
  // Maybe 'skip'
  return 'preserve';
}
```

Reply:
Hmm, indeed interesting. So there are actually 3 different actions loader could take: leave file as is (just ignore it), remove sourceMappingURL (for the devtools sake), or process sourcemaps with the removal of sourceMappingURL (current behavior) ?

I see how removing sourceMappingURL will improve the DX by clearing out the devtools warnings. 
What if we will always remove the sourceMappingURL even if we skip the resource? This will make the loader less flexible though...

Reply:
Keeping `sourceMappingURL` is useful for packages with absolute source map URLs on CDN or something else (rare case, but we should consider this).

Everything that the `skipSourceMappingURL` function returns means do not handle, so `remove` and `preserve` should be enough I think.

But there is one more case here - fixing broken URLs (also rare case), but also can be useful sometimes.